AsyncDisplayKit/ASImageNode.mm

```diff
@@ -110,17 +109,11 @@ - (instancetype)initWithViewBlock:(ASDisplayNodeViewBlock)viewBlock didLoadBlock
   return nil;
 }
 
-- (void)dealloc
```

Comment:
Although I haven't tracked it down exactly, I strongly suspect this is probably an artifact of the branch that I originally made these changes too (332ab23), which was current at the time.  Due to the nature in which we tried to get this change in (e.g., I sent you the file as-is directly), this context was lost as a result.

I believe the correct thing to do is remove this part of the diff - restore the `-dealloc` code removed - and verify that the additional `-dealloc` code that is "apparently" deleted here was in fact added as a result of a change / merged PR that happened some time after `332ab23`.